client/a8c-for-agencies/sections/woopayments/primary/woopayments-overview/style.scss

```diff
@@ -2,7 +2,11 @@
 @import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/variables";
 
+$woocommerce-purple-0: #F2EDFF;
+$woocommerce-purple-5: #E1D7FF;
+$woocommerce-purple-40: #873EFF;
```

Comment:
I believe we should be able to reference existing Woo colors using --studio-woocommerce-purple-* variables.

e.g.

```
 var( --studio-woocommerce-purple-5 );
```